src/lib/services/users.ts

```diff
@@ -63,9 +65,43 @@ export default class Users {
             id: party.id,
           },
         },
+        name: username,
+        avatar_url: Users.generateImage(username),
       },
     });
-
     return party;
   }
+
+  private static generateImage(userName: string): string {
+    /*
+     * GET Options
+     * r : radius of icon 
+     * m : margin of icon
+     * b : background color - 24 bit rgb
+     * w : width of icon
+     * h : height of icon
+     * colorLevel : brightness of color
+     * 
+     * EX: ?r=4&m=2&b=%23b99d9d&w=1&h=1&colorLevel=800
+     */
+    const options = ""; // I'm leaving this blank for now, but we'll want to add options so the image fits nicely in the chat window
+    const baseUrl = "https://avatars.dicebear.com/api/identicon/";
+    return baseUrl + encodeURIComponent(userName) + ".svg" + options;
+  }
```

Comment:
Let's use `stringifyUrl` and pass in query params as an object. Also, we can leave a link to the options.